### PR TITLE
fix: expose OSD for virtual FC to pick up

### LINKED ISSUE #3023: OSD broken in Virtual FC since moved to modules
### Describe the bug

The OSD does not work for the Virtual FC. It gives a javascript error because the OSD variable does not exist:

https://github.com/betaflight/betaflight-configurator/blob/db99a988073a5f0f6c87ec00aa5d31b6c618b7a4/src/js/VirtualFC.js#L178-L215

The `OSD` can be changed to `TABS.osd`, but it seems now it can't access to the `data` member neither. I'm not too sure what is the best way to fix that.

I think it was broken since https://github.com/betaflight/betaflight-configurator/pull/2464
@chmelevskij maybe you can look at it?

### To Reproduce

- Start Betaflight Configurator
- Connect with the Virtual FC
- Go to the Configuration tab and enable the OSD
- Go to the OSD tab. It will not load.

### Expected behavior

To load the OSD tab.

### Configurator version

10.9.0-debug-185db5b

### Flight controller configuration

```txt
No matter, virtual one.
```

### Add any other context about the problem that you think might be relevant here

Nothing

### PR DESCRIPTION
**Pull request by chmelevskij**
Closes #3023

### CODE REVIEW

**Review by Asizon — APPROVED** [review]

File: src/js/tabs/osd.js [hunk 1]
```diff
@@ -3325,6 +3325,7 @@ osd.cleanup = function(callback) {
 };
 
 window.TABS.osd = osd;
+window.OSD = OSD;
```
Comment (McGiverGim) [inline src/js/tabs/osd.js:3328]: Maybe is better to modify in the VirtualFC file:
https://github.com/betaflight/betaflight-configurator/blob/db99a988073a5f0f6c87ec00aa5d31b6c618b7a4/src/js/VirtualFC.js#L179

Modify `const virtualOSD = OSD` by `const virtualOSD = TABS.osd`? In this way we don't duplicate here the variable. I did a little test yesterday but it not worked, but maybe I did something wrong.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/osd.js (hunk 1 above)
Comment (chmelevskij) [reply src/js/tabs/osd.js:3328]: Good shout. Will grab a look

Comment (haslinghuis) [reply src/js/tabs/osd.js:3328]: Can we use export / import?

Comment (chmelevskij) [reply src/js/tabs/osd.js:3328]: Eventually we can, but would need move virtual fc to modules first. But at the moment I'm trying to do similar thing to GUI first.

**Review by limonspb — APPROVED** [review]

File: src/js/tabs/osd.js (hunk 1 above)
Comment (haslinghuis) [reply src/js/tabs/osd.js:3328]: @chmelevskij any update? Do you like to have this merged as temporary fix?

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**